.travis.yml

```diff
@@ -1,24 +1,16 @@
 language: go
-sudo: required
-services:
-  - docker
-go:
-  - 1.8.x
```

Comment:
Set a travis go version that is at least 1.11 (for GO111MODULE). Everything else looks good.

Reply:
will do!